### PR TITLE
Update RHEL installation guide to reflect latest repo changes

### PR DESCRIPTION
**Pull request by vutny**
This PR updates docs with relevant info mentioned in issue #28142

### CODE REVIEW

**Comment by cachedout** [comment]
@terminalmage Could I get you to take a look here as well?

File: doc/topics/installation/rhel.rst [hunk 1]
```diff
@@ -149,7 +160,7 @@ Installing from ``epel-testing``
 ********************************
 
 When a new Salt release is packaged, it is first admitted into the
-``epel-testing`` repository, before being moved to the stable repo.
+``epel-testing`` repository, before being moved to the main stable repository.
```
Comment (terminalmage) [inline doc/topics/installation/rhel.rst]: "main stable repository" is redundant, if anything this should be changed to "stable epel repository".

File: doc/topics/installation/rhel.rst [hunk 2]
```diff
@@ -84,24 +84,36 @@ To install using the SaltStack repository:
    - ``yum install salt-cloud``
 
 .. note::
-    As of 2015.8.4, EPEL support is no longer required when installing on Red Hat 5. (EPEL
-    support was previously required on Red Hat 5, but not on Red Hat 6 or 7).
+    As of 2015.8.4, EPEL repository is no longer required for installing on
```
Comment (terminalmage) [inline doc/topics/installation/rhel.rst]: Actually, this should be 2015.8.0.

**Comment by vutny** [comment]
@terminalmage Thanks for the feedback! Corrected according to your comments.

**Pull request merged**